markdown/guide/what-are-engines.md

```diff
@@ -1,27 +1,31 @@
-## What Are Engines?
+## What are Engines?
 
-While a lot of developers may have heard the term "Engine", there is often a lot of unfamiliarity with what that term actually means. So, let us take a moment and make sure we are all on the same page before diving into building an Engine.
+Engines can be considered miniature applications that provide functionality to their host applications. Engines are isolated, `composable applications`, they have almost all the same features as normal Ember applications, except an [Engine](https://api.emberjs.com/ember/release/classes/Engine) requires a host application to boot it and provide a Router instance.
 
-### What Engines Are Conceptually
+"Engines allow multiple logical applications to be composed together into a single application from the user's perspective." - [Engines Guides](http://ember-engines.com/)
 
-The summary from the [Engines RFC](https://github.com/emberjs/rfcs/blob/master/text/0010-engines.md) states that Engines are constructs that:
+The often used example of this is a "blogging" application. While a blog itself could be a full-fledged application, you may also wish to use it as a specific subsection of your overall application. If you threw a "news feed" engine into that mix, you might get an application structure that looks something like this:
 
-> ...allow multiple logical applications to be composed together into a single application from the user's perspective.
+![Diagram of Application with two Engines](../images/02-app-diagram.png)
 
-The key term here is "logical application". That is, an Engine represents a set of functionality and user experiences that could logically be considered an application. However, they differ from normal Ember applications in that they are intended to be composed within a parent application in order to create a single, cohesive user experience.
+Even though the application has sets of functionality represented by Engines, there is room for some additional functionality that the host application itself provides.
 
-The oft used example of this is a "blogging" application. While a blog itself could be a full-fledged application, you may also wish to use it as a specific subsection of your overall application. If you threw a "news feed" engine into that mix, you might get an application structure that looks something like this:
+## Why use Engines?
 
-![Diagram of Application with two Engines](../images/02-app-diagram.png)
+Large companies are increasingly adopting Ember.js to power their entire product lines. Often this means separate teams (sometimes distributed around the world) working on the same app. Typically, responsibility is shared by dividing the application into one or more "sections". How this division is actually implemented varies from team to team. 
```

Comment:
We need to avoid assuming "companies" and think in terms of "organizations".

```suggestion
Large organizations often use Ember.js to power sophisticated web applications. These apps may require collaboration among several teams, sometimes distributed around the world. Typically, responsibility is shared by dividing the application into one or more "sections". How this division is actually implemented varies from team to team. 
```